src/audio/dai-zephyr.c

```diff
@@ -633,7 +633,7 @@ static int dai_set_sg_config(struct dai_data *dd, struct comp_dev *dev, uint32_t
 				 period_count);
 			buf_size = period_count * period_bytes;
 			do {
-				if (IS_ALIGNED(buf_size, max_block_count)) {
+				if (SOF_IS_ALIGNED(buf_size, max_block_count)) {
```

Comment:
is it the plan then after https://github.com/zephyrproject-rtos/zephyr/pull/67243 is merged to move to the "proper" Zephyr-provided `IS_ALIGNED()` and to rename back?

Reply:
I think an alternative would be to take the Zephyr change with IS_ALIGNED in, then add IS_ALIGNED definition only for xtos builds in SOF, and call it a day.

The current ifdef can carry us until then.

Or we merge this, but I agree this is going against our long term plans. We want to use native Zephyr interfaces in generic SOF code whenever possible (e.g. #5794).